### ISSUE TITLE
check_random_url fails, I don't know why

### ISSUE THREAD

**Issue by fflorent** [yaal-coop/scim2-tester#8]
I would need help to understand what is expected for `check_random_url`, I cannot figure it out myself.

My SCIM endpoint returns this payload on when querying non-existent endpoint. As you can see, it answers with a 404 status code and there is an error payload. Is there anything that doesn't conform to the spec?

```bash
curl -H 'Authorization: Bearer api_key_admin' -H 'Content-Type: application/json' http://localhost:3000/scim/whatever -v | jq
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0*   Trying 127.0.0.1:3000...
* Connected to localhost (127.0.0.1) port 3000 (#0)
> GET /scim/whatever HTTP/1.1
> Host: localhost:3000
> User-Agent: curl/7.88.1
> Accept: */*
> Authorization: Bearer api_key_admin
> Content-Type: application/json
> 
< HTTP/1.1 404 Not Found
< X-Powered-By: Express
< Content-Type: application/scim+json; charset=utf-8
< Content-Length: 104
< ETag: W/"68-zh+4yIoz1govWdaghCJSScM0qgQ"
< Date: Sun, 01 Sep 2024 11:25:08 GMT
< Connection: keep-alive
< Keep-Alive: timeout=5
< 
{ [104 bytes data]
100   104  100   104    0     0   7679      0 --:--:-- --:--:-- --:--:--  8000
* Connection #0 to host localhost left intact
{
  "schemas": [
    "urn:ietf:params:scim:api:messages:2.0:Error"
  ],
  "status": "404",
  "detail": "Endpoint Not Found"
}
```

**Comment by azmeuk** [comment]
Fixed and release in 0.1.5. This was due to the recent update to scim2-models. I wish I had written the unit test suite earlier...

`check_random_url` simply checks that an invalid URL returns a 404 HTTP code with a SCIM Error object payload in json format.

**Comment by fflorent** [comment]
Thank you so much!
